### ISSUE TITLE
Version 0.6.0 is not tagged in git.

### ISSUE THREAD

**Issue by roblabla** [tikv/jemallocator#98]
It'd be nice to get a git tag for version 0.6.0, to make it easier to track the diffs between versions.

**Comment by BusyJay** [comment]
Indeed, I was waiting #96. The tag will be tagged on the commits with title bump version xxx.
